### PR TITLE
fix: pass previewUrlCallback function to the global scope

### PR DESCRIPTION
**Pull request by nd0ut**

### CODE REVIEW

File: src/uploader.js [hunk 1]
```diff
@@ -80,6 +84,7 @@ const useWidget = (
       if (locale) delete window.UPLOADCARE_LOCALE
       if (localePluralize) delete window.UPLOADCARE_LOCALE_PLURALIZE
       if (localeTranslations) delete window.UPLOADCARE_LOCALE_TRANSLATIONS
+      if (previewUrlCallback) delete window.UPLOADCARE_PREVIEW_URL_CALLBACK
     }
   }, [locale, localeTranslations, localePluralize])
```
Comment (FilippovAM) [inline src/uploader.js:89]: use hook useCallback for set function in deps.
previewUrlCallback

Comment (nd0ut) [reply src/uploader.js]: 😨 fixed

**Review by LittleCreamSoda — APPROVED** [review]

**Comment by nd0ut** [comment]
@shipjs prepare

**Comment by nd0ut** [comment]
@shipjs prepare

**Comment by nd0ut** [comment]
@shipjs prepare

**Pull request merged**